app/ts/Workspace/WorkspaceContext.tsx

```diff
@@ -50,6 +51,9 @@ export function WorkspaceContextProviderComponent({
     }
   }, [scales, oldWorkspace]);
   const workspace: IWorkspace = buildWorkspace(oldWorkspace, workspaceId);
+  const subproblemPvfs: Record<string, IPvf> = buildSubproblemPvfs(
+    oldWorkspace.problem.criteria
+  );
```

Comment:
This, to me,  seems like the wrong context to put this. Putting this inside the subproblemcontext makes more sense.